dGame/dMission/Mission.cpp

```diff
@@ -47,6 +47,10 @@ Mission::Mission(MissionComponent* missionComponent, const uint32_t missionId) {
 
         m_Tasks.push_back(task);
     }
+
+    auto tableData = CDClientDatabase::ExecuteQuery("SELECT LevelCapCurrencyConversion FROM WorldConfig WHERE WorldConfigID = 1 LIMIT 1;");
+    m_CurrencyConversionRate = tableData.getIntField(0, -1);
+    tableData.finalize();
```

Comment:
Same here, cache this query. This will run a lot of times, and as it will always have the same result it's redundant.

Reply:
I was thinking of doing this query when a new MissionComponent is called and storing the result in there, then passing it to the constructor of Mission.  Would this be the way to go about the change or is there a better place to store this and access the variable?

Reply:
For now I am going to cache this in the same file I am caching the max level (dZoneManager.cpp).